frontend/src/components/shared/BuyerNavbar.tsx

```diff
@@ -2,17 +2,22 @@ import * as React from "react"
 import { Button } from "@chakra-ui/react";
 import "./Navbar.css"
 import { useNavigate } from "react-router-dom";
+import huskyIcon from '../../assets/huskyIcon.png';
+import { FaBox, FaStore } from "react-icons/fa";
 const BuyerNavbar: React.FC<{}> = (props) => {
     const navigate = useNavigate();
 
     return (
         <div className="navbar">
             <h3 className="clickableText" onClick={() => {
                 navigate("/")
-            }}>Hersey Spiritwear (Buyer)</h3>
+            }}>
+                <img src={huskyIcon} alt="Husky Icon" className="image"/>
```

Comment:
Icon is blurry right now:
![image](https://github.com/miguelaenlle/merch-order-request-form/assets/66172204/70bb6d32-02b1-4082-8fe1-0af2f325b138)

Suggestion: Import and use `JHHS_LOGO`, which is a link to a high-res logo
```suggestion
                <img src={JHHS_LOGO} alt="Husky Icon" className="image"/>
```
Then, play with the padding & sizing to get the icon's spacing and sizing to match the design on Notion.